<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="chrome=1"><meta name=viewport content="width=device-width,initial-scale=1"><meta name=robots content="noodp"><link rel=next href=https://trangle1302.github.io/research/><link rel=canonical href=https://trangle1302.github.io/publications/><link rel="shortcut icon" type=image/x-icon href=/favicon.ico><link rel=apple-touch-icon sizes=180x180 href=/apple-touch-icon.png><link rel=icon type=image/png sizes=32x32 href=/favicon-32x32.png><link rel=icon type=image/png sizes=16x16 href=/favicon-16x16.png><link rel=manifest href=/site.webmanifest><link rel=mask-icon href=/safari-pinned-tab.svg color=#5bbad5><meta name=msapplication-TileColor content="#da532c"><meta name=theme-color content="#ffffff"><title>Publications | Trang Le</title><meta name=title content="Publications | Trang Le"><link rel=stylesheet href=/font/iconfont.css><link rel=stylesheet href=/css/main.min.css><meta name=twitter:card content="summary"><meta name=twitter:title content="Publications"><meta name=twitter:description content="Ouyang, W., Le, T., Xu, H., and Lundberg, E. Interactive biomedical segmentation tool powered by deep learning and ImJoy. F1000Research (2021). https://doi.org/10.12688/f1000research.50798.1
  Mahdessian, D., Cesnik, A.J., Gnann, C. et al. Spatiotemporal dissection of the cell cycle with single-cell proteogenomics. Nature 590, 649–654 (2021). https://doi.org/10.1038/s41586-021-03232-9
  Zhernakova, D. V., Le, T. H., Kurilshikov, A., Atanasovska, B., Bonder, M. J., Sanna, S., Claringbould, A., Võsa, U., Deelen, P."><script type=application/ld+json>{"@context":"http://schema.org","@type":"BlogPosting","headline":"Publications","mainEntityOfPage":{"@type":"WebPage","@id":"https:\/\/trangle1302.github.io\/publications\/"},"genre":"page","wordcount":196,"url":"https:\/\/trangle1302.github.io\/publications\/","datePublished":"2021-06-24T00:00:00\u002b00:00","dateModified":"2021-06-24T00:00:00\u002b00:00","author":{"@type":"Person","name":"map[name:Trang Le]"},"description":""}</script></head><body><div class=wrapper><nav class=navbar><div class=container><div class="navbar-header header-logo"><a href=https://trangle1302.github.io/>Trang Le</a></div><div class="menu navbar-right"><a class=menu-item href=/about title>About</a>
<a class=menu-item href=/research title>Research</a>
<a class=menu-item href=/publications title>Publications</a>
<a href=javascript:void(0); class=theme-switch><i class="iconfont icon-sun"></i></a>&nbsp;</div></div></nav><nav class=navbar-mobile id=nav-mobile style=display:none><div class=container><div class=navbar-header><div><a href=javascript:void(0); class=theme-switch><i class="iconfont icon-sun"></i></a>&nbsp;<a href=https://trangle1302.github.io/>Trang Le</a></div><div class=menu-toggle><span></span><span></span><span></span></div></div><div class=menu id=mobile-menu><a class=menu-item href=/about title>About</a>
<a class=menu-item href=/research title>Research</a>
<a class=menu-item href=/publications title>Publications</a></div></div></nav><main class=main><div class=container><div class="post-warp archive"><h2 class=post-title style=text-align:right;padding-bottom:2em>Publications</h2><div class=post-content><ul><li><p>Ouyang, W., Le, T., Xu, H., and Lundberg, E. Interactive biomedical segmentation tool powered by deep learning and ImJoy. F1000Research (2021). <a href=https://doi.org/10.12688/f1000research.50798.1>https://doi.org/10.12688/f1000research.50798.1</a></p></li><li><p>Mahdessian, D., Cesnik, A.J., Gnann, C. et al. Spatiotemporal dissection of the cell cycle with single-cell proteogenomics. Nature 590, 649–654 (2021). <a href=https://doi.org/10.1038/s41586-021-03232-9>https://doi.org/10.1038/s41586-021-03232-9</a></p></li><li><p>Zhernakova, D. V., Le, T. H., Kurilshikov, A., Atanasovska, B., Bonder, M. J., Sanna, S., Claringbould, A., Võsa, U., Deelen, P., Franke, L., de Boer, R. A., Kuipers, F., Netea, M. G., Hofker, M. H., Wijmenga, C., Zhernakova, A., Fu, J., LifeLines cohort study, & BIOS consortium (2018). Individual variations in cardiovascular-disease-related protein levels are driven by genetics and gut microbiome. Nature genetics, 50(11), 1524–1532. <a href=https://doi.org/10.1038/s41588-018-0224-7>https://doi.org/10.1038/s41588-018-0224-7</a></p></li><li><p>Kolesnikova, T. A, Kiragosyan, G., Le, T. H. N, Springer, S., & Winterhalter, M. (2017). Protein A Functionalized Polyelectrolyte Microcapsules as a Universal Platform for Enhanced Targeting of Cell Surface Receptors. ACS Applied Materials & Interfaces, 9, 11506-11517. doi: 10.1021/acsami.7b01313</p></li></ul><h1 id=posterstalks>Posters/Talks</h1><ul><li>Science for Life Laboratory seminar, online, June 2021. Short talk: “Modelling the spatial cell cycle with deep learning”</li><li>AICS/HPA/CZI/CZB symposium, online, February 2021. Short talk: “Kaggle challenge: HPA - Single Cell Classification”</li><li>NEUBIAS, Bordeaux, March 2020. Poster presentation: “Deep learning model for the cell cycle”</li></ul></div></div></div></main><footer class=footer><div class=copyright>&copy;
<span itemprop=copyrightYear>2021 - 2021</span>
<span class=author itemprop=copyrightHolder><a href=https://trangle1302.github.io/>lengochaitrang@gmail.com</a> |</span>
<span>Crafted with ❤️ by <a href=https://github.com/Fastbyte01/KeepIt target=_blank rel="external nofollow noopener noreffer">KeepIt</a> & <a href=https://gohugo.io/ target=_blank rel="external nofollow noopener noreffer">Hugo</a></span></div></footer><script src=/js/vendor_no_gallery.min.js async></script></div></body></html>